Src/NuGet.Config

```diff
@@ -1,6 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<configuration>
-  <activePackageSource>
```

Comment:
This setting has been deprecated since Nuget 3.x and it doesn't seem to do anything since Visual Studio 2017.